app/grandchallenge/algorithms/templates/algorithms/data_tables/result_list.html

```diff
@@ -0,0 +1,90 @@
+{% load url %}
+{% load profiles %}
+{% load workstations %}
+{% load guardian_tags %}
+{% load pathlib %}
+{% load humanize %}
+{% load evaluation_extras %}
+
+{{ result.job.created|naturaltime }}
+<td>
+    {{ result.job.creator|user_profile_link }}
+<td>
+    <a href="#resultInfoModal"
+       class="badge badge-primary"
+       data-toggle="modal"
+       data-target="#resultInfoModal"
+       data-title="Result Info"
+       data-output="{% firstof result.output 'No output produced.' %}"
+       data-pk="{{ result.pk }}"
+       title="Result Info">
+        <i class="fa fa-file"></i> {{ result.job.image.name }}
+    </a>
+<td>
```

Comment:
How would we add `data-order` to this column? At the moment, sorting by visibility (in production) does not work.

Reply:
The ordering is handled in the view. Datatables just sends an index for the column to sort on, so the columns need to coincide with what is defined in the view. Visibility sorting is working I think, but I'll check to be sure.

Reply:
Yes, but `data-order` tells data tables how to sort, so it's important to set. Without it, it will sort on the contents of the cell which are human readable, so goes completely wrong for dates and doesn't work for visibility right now as data tables does not know how to sort icons. This is why we had `data-order="{{ result.job.created|date:"U" }}"` so that it can sort on a unixtime, but display `{{ result.job.created|naturaltime }}` a natural time for the user.

Reply:
The sorting is now done on the database using the values defined in the `columns` attribute in the view. So the `data-order` is not used anymore.

Reply:
Oh, I see! Even better.